libs/sdk-ui-dashboard/src/model/commandHandlers/widgets/modifyDrillForInsightWidgetHandler.ts

```diff
@@ -0,0 +1,80 @@
+// (C) 2021 GoodData Corporation
+
+import { DashboardContext } from "../../types/commonTypes";
+import { ModifyDrillsForInsightWidget } from "../../commands";
+import { SagaIterator } from "redux-saga";
+import { put, select } from "redux-saga/effects";
+
+import { DashboardInsightWidgetDrillsModified, insightWidgetDrillsModified } from "../../events/insight";
+import { selectWidgetsMap } from "../../state/layout/layoutSelectors";
+import { validateExistingInsightWidget } from "./validation/widgetValidations";
+import { InsightDrillDefinition } from "@gooddata/sdk-backend-spi";
+import { selectDrillTargetsByWidgetRef } from "../../state/drillTargets/drillTargetsSelectors";
+import { existsDrillDefinitionInArray } from "../../../_staging/drills/InsightDrillDefinitionUtils";
+import { validateInsightDrillDefinition } from "./validation/insightDrillDefinitionValidation";
+import { layoutActions } from "../../state/layout";
+
+export function* modifyDrillForInsightWidgetHandler(
```

Comment:
Please rename this and the file to reflect the new name of the command (modify drills instead of modify drill).

Reply:
Done